tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -626,6 +629,115 @@ void test_spacetime_derivative_of_spacetime_metric(
   }
 }
 
+// Test GeneralizedHarmonic::ricci_tensor
+void test_spatial_ricci_tensor(
+    const size_t grid_size_each_dimension,
+    const std::array<double, 3>& lower_bound,
+    const std::array<double, 3>& upper_bound) noexcept {
+  using frame = Frame::Inertial;
+  constexpr size_t VolumeDim = 3;
+  // Setup grid
+  Mesh<VolumeDim> mesh{grid_size_each_dimension, Spectral::Basis::Legendre,
+                       Spectral::Quadrature::GaussLobatto};
+  const auto coord_map =
+      domain::make_coordinate_map<Frame::Logical, Frame::Inertial>(Affine3D{
+          Affine{-1., 1., lower_bound[0], upper_bound[0]},
+          Affine{-1., 1., lower_bound[1], upper_bound[1]},
+          Affine{-1., 1., lower_bound[2], upper_bound[2]},
+      });
+
+  // Setup coordinates
+  const auto x_logical = logical_coordinates(mesh);
+  const auto x = coord_map(x_logical);
+  const Direction<VolumeDim> direction(1, Side::Upper);  // +y direction
+  const size_t slice_grid_points =
+      mesh.extents().slice_away(direction.dimension()).product();
+  const auto inertial_coords = [&slice_grid_points, &lower_bound]() {
+    tnsr::I<DataVector, VolumeDim, frame> tmp(slice_grid_points, 0.);
+    // +y direction
+    get<1>(tmp) = 0.5;
+    for (size_t i = 0; i < VolumeDim; ++i) {
+      for (size_t j = 0; j < VolumeDim; ++j) {
+        get<0>(tmp)[i * VolumeDim + j] =
+            lower_bound[0] + 0.5 * static_cast<double>(i);
+        get<2>(tmp)[i * VolumeDim + j] =
+            lower_bound[2] + 0.5 * static_cast<double>(j);
+      }
+    }
+    return tmp;
+  }();
+
+  auto local_inverse_spatial_metric =
+      make_with_value<tnsr::II<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_phi =
+      make_with_value<tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_d_pi =
+      make_with_value<tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_d_phi =
+      make_with_value<tnsr::ijaa<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+
```

Comment:
Comment here that these values are chosen to reproduce a result from SpEC

Reply:
added